File: Palindrome.java
package My_Works;
import java.util.Scanner;
public class Palindrome 
{
	public static void main(String[] args)
	{
		System.out.println("Enter a Word: ");
		Scanner sc = new Scanner(System.in);
		String Word = sc.next();
		
		StringBuffer ReversedWord = new StringBuffer(Word);
		ReversedWord.reverse();
		
		if(ReversedWord.toString().equalsIgnoreCase(Word))
		{
			System.out.println("The given word is Palindrome!");
		}
		else
		{
			System.out.println("The given word is not a Palindrome!");
		}
	}
}

File: Palindrome.py
num = int(input("Enter a number: "))
copy = num
a = 0
b = 0
while copy!=0:
    a = copy%10
    b = (b*10)+a
    copy = copy//10
if num==b:
    print("The given number is a Palindrome")
else:
    print("The given number is not a Palindrome")